clcache.py

```diff
@@ -7,6 +7,7 @@
 # root directory of this project.
 #
 from ctypes import windll, wintypes
+import cProfile
```

Comment:
Probably, it because after first several iterations it goes to cache (system or disk cache).